python/mxnet/ranking_metrics.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
I don't think it's a good idea adding this as a standalone file under `python/mxnet`. Is it more reasonable to put integrate this metric in `python/mxnet/metric.py` file?